modules/@apostrophecms/doc-type/index.js

```diff
@@ -1059,17 +1060,26 @@ module.exports = {
         const existing = await actionModule.findForEditing(toReq, {
           _id: toId
         }).permission('view').toObject();
+
+        const eventOptions = {
+          source: draft.aposLocale.split(':')[0],
+          target: toLocale,
+          existing: Boolean(existing)
+        };
+
         // We only want to copy schema properties, leave non-schema
         // properties of the source document alone
-        const data = Object.fromEntries(Object.entries(draft).filter(([ key, value ]) => self.schema.find(field => field.name === key)));
+        const data = Object.fromEntries(Object.entries(draft)
+          .filter(
+            ([ key, value ]) => key === 'type' || self.schema.find(field => field.name === key)
```

Comment:
Needed because we want the type to be available in the beforeLocalize event. Spreading the object is not an option as we want the modules to allow mutation the document in the pre-save event.